starter_code/app.js

```diff
@@ -11,11 +11,50 @@ app.set('views', __dirname + '/views');
 app.use(express.static(path.join(__dirname, 'public')));
 
 
+hbs.registerPartials(__dirname + '/partials')
+
+
 
 app.get('/', (req, res, next) => {
-  res.render('index');
+  let title = "IronBeers"
+  res.render("index",{
+    title
+  });
+});
+
+app.get('/beers', (req, res, next) => {
+  let title = "Beers";
+  punkAPI.getBeers()
+  .then(beers => {
+    res.render("beers",{
+      title,
+      beers
+    });
+  })
+  .catch(error => {
+    console.log(error)
+  })
+});
+
+
+
+app.get('/randombeer', (req, res, next) => {
+  let title = "Random Beer"
+  punkAPI.getRandom()
+  .then(beers => {
+    let beer = beers[0];
+    res.render("randombeer",{
+      beer,
+      title
+    });
+  })
+  .catch(error => {
+    console.log(error)
+  })
 });
 
 
 
+
+
 app.listen(3000);
```

Comment:
Use a callback inside the app.listen with a console log in order to let you know when a server is running 😉 
`const PORT = 3000`

``` 
app.listen(PORT, ()=>{
  console.log(`listen on port: ${PORT}`)
});